DynmapCore/src/main/java/org/dynmap/hdmap/IsoHDPerspective.java

```diff
@@ -1178,6 +1178,7 @@ public IsoHDPerspective(DynmapCore core, ConfigurationNode configuration) {
 
     @Override
     public boolean render(MapChunkCache cache, HDMapTile tile, String mapname) {
+        final long startTimestamp = System.currentTimeMillis();

```

Comment:
The timestamp should not be the current time, but instead the time of the latest tile used within this zoomout tile

Reply:
This is the rendering code for the lowest level of images